wpk/generate_wpk_package.sh

```diff
@@ -15,40 +15,44 @@ LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+INSTALLATION_PATH="/var"
 
 trap ctrl_c INT
 
 function build_wpk_windows() {
   local BRANCH="$1"
   local DESTINATION="$2"
-  local KEYDIR="$3"
-  local CONTAINER_NAME="$4"
-  local JOBS="$5"
-  local PACKAGE_NAME="$6"
-  local OUT_NAME="$7"
-  local CHECKSUM="$8"
-  local CHECKSUMDIR="$9"
+  local KEYDIR="${3}"
```

Comment:
In this case, you could remove `{}`, like `KEYDIR="$3"` ?

Reply:
All unified to ${} https://github.com/wazuh/wazuh-packages/pull/408/commits/c331bd9e21fa8a17b41d971cd0348c35bdd54bd9